core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -216,6 +243,15 @@ private void setTelemetryProcessors(ApplicationInsightsXmlConfiguration appConfi
         }
     }
 
+    private void addDefaultBuiltInProcessors(List<String> p) {
+        p.add(FixedRateSamplingTelemetryProcessor.class.getCanonicalName());
```

Comment:
maybe I should use the FQCNs here....

Reply:
I am not sure if it causes significant over head. FQCNs are difficult to understand. My focus would be to keep this as it is and focus on simplicity :) . Please also add java doc comments even though method is private with short description for better understanding.

Reply:
I'm asking myself this because these are "depedencies" of this class and clutter the import statements, IMO.  It doesn't really matter, though.